cmd/gazelle/fix-update.go

```diff
@@ -243,15 +243,19 @@ func runFixUpdate(cmd command, args []string) (err error) {
 	mrslv := newMetaResolver()
 	kinds := make(map[string]rule.KindInfo)
 	loads := genericLoads
+	var crossResolvers []resolve.CrossResolver
 	for _, lang := range languages {
 		cexts = append(cexts, lang)
 		for kind, info := range lang.Kinds() {
 			mrslv.AddBuiltin(kind, lang)
 			kinds[kind] = info
 		}
 		loads = append(loads, lang.Loads()...)
+		if c, ok := lang.(resolve.CrossResolver); ok {
+			crossResolvers = append(crossResolvers, c)
+		}
```

Comment:
We shouldn't need to check the `CrossResolver` interface here. Let's just put all extensions in a `[]interface{}`:

```
exts := make([]interface{}, 0, len(languages))
for _, lang := range languages {
  ...
  exts = append(exts, lang)
}
```

In hindsight, I wish it were possible for authors to write extensions that don't implement `language.Language`; they could register extensions that implement any subset of supported interfaces like `config.Configurer` or `resolve.CrossResolver`.

At least in this case, we can pass extensions to `resolve` without worrying what interfaces are implemented or might be added in the future.